apps/code-examples/src/app/code-examples/indicators/help-inline/basic/help-inline-demo.component.spec.ts

```diff
@@ -5,7 +5,7 @@ import { SkyHelpInlineHarness } from '@skyux/indicators/testing';
 import { HelpInlineDemoComponent } from './help-inline-demo.component';
 import { HelpInlineDemoModule } from './help-inline-demo.module';
 
-describe('Basic help inline', async () => {
+describe('Basic help inline', () => {
```

Comment:
This was from a find by scot freeman. Copy paste error with past code example updates from other test harness stories.